src/stytch-provider.ts

```diff
@@ -72,10 +210,15 @@ function StytchProvider(this: any, options: StytchProviderOptions) {
     let project_id = res.keymap.project_id.value
     let secret = res.keymap.secret.value
 
+    const agent = new https.Agent({
+      keepAlive: true,
+    })
+
     seneca.shared.sdk = new Stytch.Client({
       project_id,
       secret,
-      env: 'live' === options.env ? Stytch.envs.live : Stytch.envs.test
+      env: 'live' === options.env ? Stytch.envs.live : Stytch.envs.test,
```

Comment:
are these environments hard-coded concepts in the stytch system?

Reply:
according to the docs: https://github.com/stytchauth/stytch-node#customizing-the-https-agent

Reply:
ok